tests/testthat/test-object_usage_linter.R

```diff
@@ -410,6 +410,18 @@ test_that("interprets glue expressions", {
       glue::glue('The answer is {local_var}.')
     }
   "), "local_var", object_usage_linter(interpret_glue = FALSE))
+
+  # call in glue is caught
+  expect_lint(
+    trim_some("
+      fun <- function() {
```

Comment:
Also add `local_unused <- identity` to make sure the test covers what is intended?